test/test_loop_rotation.cpp

```diff
@@ -15,11 +15,7 @@
 
 namespace nvfuser {
 
-class LoopRotationTest : public NVFuserTest {
- private:
-  // Please see note [Limitation of boundary assert]
-  EnableOutOfBoundAssert guard;
-};
+class LoopRotationTest : public NVFuserTest {};
```

Comment:
nitpick, just remove this and replace all `LoopRotationTest` with `NVFuserTest`.

Reply:
The main reason why class `LoopRotationTest` exists is because I want to have a different name so that it can be easily filtered out by `gtest_filter`, not because I want to enable the boundary check. Are you saying that you prefer to have all tests named as `NVFuserTest`, or you didn't mean that?